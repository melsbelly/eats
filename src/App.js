import React from 'react';
import {Route} from "react-router-dom";
import {ConnectedRouter} from "connected-react-router";
import {Home} from "./components/Home";
import {history} from "./store";

export default function App() {
    return (
        <ConnectedRouter history={history}>
            <Route exact path="/" component={Home}/>
        </ConnectedRouter>
    );
};
