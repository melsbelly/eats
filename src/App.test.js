import React from 'react';
import App from './App';

it('renders correctly', () => {
    expect(App()).toMatchSnapshot();
});
