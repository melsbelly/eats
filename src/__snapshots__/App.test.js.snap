// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Memo(ConnectFunction)
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Memo(ConnectFunction)>
`;
